releases/4.0.3.md

```diff
@@ -0,0 +1,154 @@
+
++ TiKV
+
+    - A new configuration `backup.num-threads` is introduced to control the backup thread pool size. [#8199](https://github.com/tikv/tikv/pull/8199)
+    - Do not send store heartbeat when receiving snapshot [#8136](https://github.com/tikv/tikv/pull/8136)
+
++ PD
+
+    - Support json formatted log [#2565](https://github.com/pingcap/pd/pull/2565)
++ Dashboard
+    - Improved KeyViz bucket merge for cold logical ranges [#674](https://github.com/pingcap-incubator/tidb-dashboard/pull/674)
+    - Renamed config item disable-telemetry to enable-telemetry to be more consistent [#684](https://github.com/pingcap-incubator/tidb-dashboard/pull/684)
+    - Show progress bar when switching pages [#661](https://github.com/pingcap-incubator/tidb-dashboard/pull/661)
+    - Slow log search now follows the same behaviour as log search when there are space separators [#682](https://github.com/pingcap-incubator/tidb-dashboard/pull/682)
+
++ TiFlash
+
+    - Change the unit of the panels of DDL jobs in Grafana to operations per minute [#923](https://github.com/pingcap/tics/pull/923)
+    - Reduce IOPS in TiFlash proxy [#856](https://github.com/pingcap/tics/pull/856)
+
++ TiCDC
+
+    - replace table ID with table name in metrics [#695](https://github.com/pingcap/ticdc/pull/695)
+
++ BR
+
+    - Support JSON log [#336](https://github.com/pingcap/br/issues/336)
+    - Support starting pprof in-the-fly [#372](https://github.com/pingcap/br/pull/372)
+    - Speed up DDL execution by send DDL concurrently [#377](https://github.com/pingcap/br/pull/377)
+
++ TiDB Lightning
+
+    - [black-white-list] has been deprecated with a newer, easier-to-understand filter format [#332](https://github.com/pingcap/tidb-lightning/pull/332)
+
+## Others
+
++ TiDB
+
+    - Add system tables `tiflash_tables` and `tiflash_segments` [#18536](https://github.com/pingcap/tidb/pull/18536)
+    - Executor: return the OOM error when indexHashJoin worker panics [#18527](https://github.com/pingcap/tidb/pull/18527)
+    - Enable collect execution information by default [#18518](https://github.com/pingcap/tidb/pull/18518)
+    - To speed up canceling a query [#18505](https://github.com/pingcap/tidb/pull/18505)
+    - Change the default value of OOMAction to OOMActionCancel. [#18502](https://github.com/pingcap/tidb/pull/18502)
+    - Add a header for tidb_decode_plan result. [#18501](https://github.com/pingcap/tidb/pull/18501)
+    - Make the configuration checker compatible with older versions of the configuration [#18046](https://github.com/pingcap/tidb/pull/18046)
+    - Insert negative primary key into auto_random table won't trigger rebase. [#17987](https://github.com/pingcap/tidb/pull/17987)
+
++ TiKV
+
+    - Support dynamically changing the shared block cache's capacity [#8232](https://github.com/tikv/tikv/pull/8232)
+    - Fix an issue that read may get stale data during merging [#8113](https://github.com/tikv/tikv/pull/8113)
```

Comment:
@uglyengineer Please help check the notes of this `Others` section belongs to which category. Thanks!